app/models/state_file_base_intake.rb

```diff
@@ -16,13 +17,10 @@ class StateFileBaseIntake < ApplicationRecord
   belongs_to :primary_state_id, class_name: "StateId", optional: true
   belongs_to :spouse_state_id, class_name: "StateId", optional: true
   accepts_nested_attributes_for :primary_state_id, :spouse_state_id
-
-  scope :accessible_intakes, -> { all }
-  devise :timeoutable, :timeout_in => 15.minutes, :unlock_strategy => :time
+  accepts_nested_attributes_for :dependents, update_only: true
```

Comment:
the ordering of the lines was bothering me so i groups things together while i was in here